e2eTests/Test-AL-Go.ps1

```diff
@@ -121,15 +125,23 @@ if ($appSourceApp) {
 # Add Existing Test App
 Run-AddExistingAppOrTestApp @project1Param -url $sampleTestApp1 -wait -branch $branch | Out-Null
 $runs++
-MergePRandPull -branch $branch
+
+# Merge and run CI/CD + Tests
+MergePRandPull -branch $branch | Out-Null
 $runs++
 
-# Run CI/CD and wait
+# Wait for CI/CD to finish
 $run = Run-CICD -wait -branch $branch
 $runs++
 
+if ($useCompilerFolder) { 
```

Comment:
This is because navcontainerhelper sets _doNotRunTests = $true_ when _doNotPublishApps = $true_? Could you add a short comment on that?

Reply:
done